rnote-ui/src/canvaswrapper.rs

```diff
@@ -256,6 +272,16 @@ mod imp {
                     .set_propagation_phase(PropagationPhase::None);
             }
         }
+
+        fn canvas_kinetic_scrolling_update(&self) {
+            self.scroller.set_kinetic_scrolling(
+                !self.canvas.touch_drawing() && self.inertial_scrolling.get(),
+            );
+
+            // Workaround for https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5863.
+            self.scroller.queue_allocate();
```

Comment:
Do we want to keep this? I'm assuming that it didn't do anything, so we should probably remove it.

Reply:
yeah, I think so too